src/app/modules/shared/components/details-fields/details-fields.component.scss

```diff
@@ -107,3 +107,17 @@ input[type="date"]::-webkit-inner-spin-button {
 input[type="date"]::-webkit-clear-button {
   display: none;
 }
+
+.container {
+  font-size: small;
+  cursor: text;
+}
+
+.autocomplete {
+  width: 80%;
```

Comment:
We do not need this since you are defining `padding: 0em`. In addition, we do not this too:
```
.ng-autocomplete {
  width: 100%;
}
```